tests/unit/EntityMenu.spec.ts

```diff
@@ -302,35 +302,27 @@ describe('Entity Menu - Amagamate button tests', () => {
       propsData: { businessId: 'BC1234567' }
     })
 
-    expect(wrapper.find('#amalgamate-button').exists()).toBe(true)
-    expect(wrapper.find('#amalgamate-button').text()).toBe('Amalgamate')
-    expect(wrapper.find('#amalgamate-button').classes()).not.toContain('v-btn--disabled') // enabled
-    wrapper.destroy()
-  })
-
-  it('don\'t display Amalgamate button if business is historical', async () => {
-    businessStore.$state.businessInfo.state = EntityState.HISTORICAL
-
-    const wrapper = mount(EntityMenu, {
-      vuetify
-    })
+    await wrapper.find('.menu-btn').trigger('click')
 
-    expect(wrapper.find('#amalgamate-button').exists()).toBe(false)
+    expect(wrapper.find('#amalgamate-list-item').exists()).toBe(true)
+    expect(wrapper.find('#amalgamate-list-item').text()).toBe('Amalgamate')
+    expect(wrapper.find('#amalgamate-list-item').classes()).not.toContain('v-list-item--disabled') // enabled
     wrapper.destroy()
   })
 
   it('amalgamate button is disabled if not allowed', async () => {
     businessStore.setLegalType(CorpTypeCd.BC_COMPANY)
-    businessStore.$state.businessInfo.state = EntityState.ACTIVE
```

Comment:
It's still good practise to set the variables that you depend on. It's unwise to assume pre-existing data. (I think unit test execution order is not guaranteed.)